os-sso72/added/launch/openshift-x509.sh

```diff
@@ -8,69 +8,59 @@ function prepareEnv() {
 }
 
 function configure() {
-  autogenerate_keystores
+  autogenerate_https_keystore_and_rhsso_truststore
 }
 
-function autogenerate_keystores() {
+function autogenerate_https_keystore_and_rhsso_truststore() {
   # Keystore infix notation as used in templates to keystore name mapping
-  declare -A KEYSTORES=( ["https"]="HTTPS" ["jgroups"]="JGroups" )
-
-  # CLOUD-2436 Don't generate the JGroups keystore if custom
-  # JGROUPS_ENCRYPT_SECRET was provided
-  if [ -n "${JGROUPS_ENCRYPT_SECRET}" ]; then
-    unset KEYSTORES["jgroups"]
-  fi
+  declare -A KEYSTORES=( ["https"]="HTTPS" )
```

Comment:
This array looks unnecessary...

Reply:
@Crumby Great catch, Pavel! Dropped, thanks!